include/alpaka/mem/fence/MemFenceGenericSycl.hpp

```diff
@@ -63,17 +49,14 @@ namespace alpaka::trait
     template<typename TMemScope>
     struct MemFence<MemFenceGenericSycl, TMemScope>
     {
-        static auto mem_fence(MemFenceGenericSycl const& fence, TMemScope const&)
+        static auto mem_fence(MemFenceGenericSycl const&, TMemScope const&)
         {
+            /* Fences implementation in SYCL follow the ISO C++ fences logic, so they are only clearly defined if
+             * there are atomic operations surrounding them. When using the fence, ensure that this condition is met
+             * in the code.
+             */
```

Comment:
Maybe just:
```suggestion
            // Fences implementation in SYCL follow the ISO C++ fences logic
```
Or remove the comment entirely.